DigitalLearningSolutions.Web/Views/TrackingSystem/Delegates/DelegateApprovals/Index.cshtml

```diff
@@ -0,0 +1,46 @@
+﻿@inject IConfiguration Configuration
+
+@using DigitalLearningSolutions.Web.ViewModels.TrackingSystem
+@using Microsoft.Extensions.Configuration
+@model DelegateApprovalsViewModel
+
+@{
+  ViewData["Title"] = "Approve delegate registrations";
+}
+
+<div class="nhsuk-grid-row">
+  <div class="nhsuk-grid-column-full">
+    <h1 class="nhsuk-heading-xl">Approve delegate registrations</h1>
+    @if (Model.Delegates.Any()) {
+      @foreach (var delegateApproval in Model.Delegates) {
+        <partial name="_unapprovedDelegateExpandable" model="delegateApproval"/>
+      }
+      <div class="nhsuk-grid-row">
+        <div class="nhsuk-grid-column-full">
+          <a class="nhsuk-button" href="" role="button">
+            Approve All
+          </a>
+        </div>
+      </div>
+    } else {
+      <div class="nhsuk-grid-row">
+        <div class="nhsuk-grid-column-full">
+          <p>No accounts need approval.</p>
+        </div>
+      </div>
+      <div class="nhsuk-grid-row">
+        <div class="nhsuk-grid-column-full">
+          <div class="nhsuk-action-link">
+            <a class="nhsuk-action-link__link" href="@Configuration["CurrentSystemBaseUrl"]/tracking/delegates">
+              <svg class="nhsuk-icon nhsuk-icon__arrow-right-circle" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-hidden="true">
+                <path d="M0 0h24v24H0z" fill="none"></path>
+                <path d="M12 2a10 10 0 0 0-9.95 9h11.64L9.74 7.05a1 1 0 0 1 1.41-1.41l5.66 5.65a1 1 0 0 1 0 1.42l-5.66 5.65a1 1 0 0 1-1.41 0 1 1 0 0 1 0-1.41L13.69 13H2.05A10 10 0 1 0 12 2z"></path>
+              </svg>
+              <span class="nhsuk-action-link__text">Manage approved delegates</span>
+            </a>
+          </div>

```

Comment:
Ah just realised we can't use a VC due to the href.. I guess we can change it when we create the delegates page :)